examples/ember-cli/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "ember-example",
+  "version": "4.0.0-alpha.23",
+  "private": true,
+  "scripts": {
+    "build": "ember build",
+    "build-storybook": "yarn build && cp -r public/* dist; build-storybook -s dist",
+    "dev": "ember serve",
+    "storybook": "yarn build && cp -r public/* dist; start-storybook -p 9009 -s dist"
```

Comment:
why should it be built before using with dev mode?

Reply:
In order to build a storybook you need an ember app to introspect on. It is because of the dynamic nature of ember where as react has an implicit import structure, you can build a component standalone, ember needs to register all components beforehand. Which is why we have the build before dev necessity.